toolchain/syno-geminilake-7.0/digests

```diff
@@ -0,0 +1,2 @@
+geminilake-gcc730_glibc226_x86_64-GPL.txz SHA1 6872bddab01e63d07568be8ad5e96dee15aa897e
+geminilake-gcc730_glibc226_x86_64-GPL.txz MD5 710dd6b43262a9de8f5cfd5341b5a314
```

Comment:
another left over still lacking SHA256

Reply:
I will fix this.
Is is as easy as `make clean && make digests` in the syno-*7.0 toolchain folders...

Reply:
Thanks @hgy59 :)